persistence/rest/pom.xml

```diff
@@ -18,8 +18,8 @@
          <artifactId>infinispan-core</artifactId>
       </dependency>
       <dependency>
-         <groupId>org.apache.httpcomponents</groupId>
-         <artifactId>httpclient</artifactId>
+         <groupId>io.netty</groupId>
+         <artifactId>netty-all</artifactId>
```

Comment:
@slaskawi I see Netty only talks about `netty-all` in their Downloads page, so I guess this is fine.